src/pages/Index.vue

```diff
@@ -884,11 +924,15 @@
 		}
 
 		.point {
-			width          : 30%;
-			max-width      : 260px;
+			/*width          : 30%;*/
+			/*max-width      : 260px;*/
```

Comment:
The width is too wide on the desktop, and since there are no margin it's difficult to see where the element A line stops and where the element B line starts